examples/ldtk/ldtk.rs

```diff
@@ -174,6 +175,11 @@ pub fn process_loaded_tile_maps(
                             y: tileset.tile_grid_size as f32,
                         };
 
+                        let spacing = TilemapSpacing {
```

Comment:
The ldtk level being loaded in does not have any existing spacing. I think this is fine to add into your own code but I'm hesitant to fix an issue that doesn't exist for the example. The example is meant to be a minimalistic example on how to load a LDTK map.

Reply:
Example code is most commonly copied directly into user code to get something working quickly, and then modified accordingly. In the current version of the example the spacing isn't even mentioned, and thus can lead to believe that the crate doesn't support spacing at all. I think it's a good addition, even if it does nothing for the example itself.